src/components/ProjectSettingsTabs/NotificationsTab/types.ts

```diff
@@ -0,0 +1,14 @@
+import { FormDataProps } from "components/SpruceForm";
+
+export interface FormState extends FormDataProps {
```

Comment:
I realized that this `extends FormDataProps` actually resulted in less strict typing so no need for it anymore!
```suggestion
export interface FormState {
```